.github/workflows/expire.yml

```diff
@@ -29,11 +25,13 @@ jobs:
           result-encoding: 'string'
           retries: 3
           script: |-
+            const repoOwner = String('${{ inputs.repo }}').split('/')[0]
```

Comment:
Optional, if you feel like it: check that `${{ inputs.repo }}` contains a `/` and fail with a nice error messaage if not.

Reply:
Thanks, updated!